[2023-11-13T10:23:11.333+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: people.save_people_to_excel manual__2023-11-13T10:23:03.264511+00:00 [queued]>
[2023-11-13T10:23:11.350+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: people.save_people_to_excel manual__2023-11-13T10:23:03.264511+00:00 [queued]>
[2023-11-13T10:23:11.351+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-11-13T10:23:11.352+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-11-13T10:23:11.353+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-11-13T10:23:11.380+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): save_people_to_excel> on 2023-11-13 10:23:03.264511+00:00
[2023-11-13T10:23:11.392+0000] {standard_task_runner.py:55} INFO - Started process 3643 to run task
[2023-11-13T10:23:11.396+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'people', 'save_people_to_excel', 'manual__2023-11-13T10:23:03.264511+00:00', '--job-id', '626', '--raw', '--subdir', 'DAGS_FOLDER/people.py', '--cfg-path', '/tmp/tmp3qctbf4o']
[2023-11-13T10:23:11.398+0000] {standard_task_runner.py:83} INFO - Job 626: Subtask save_people_to_excel
[2023-11-13T10:23:11.487+0000] {task_command.py:389} INFO - Running <TaskInstance: people.save_people_to_excel manual__2023-11-13T10:23:03.264511+00:00 [running]> on host 06d7566f07d2
[2023-11-13T10:23:11.794+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Tetiana
AIRFLOW_CTX_DAG_ID=people
AIRFLOW_CTX_TASK_ID=save_people_to_excel
AIRFLOW_CTX_EXECUTION_DATE=2023-11-13T10:23:03.264511+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-13T10:23:03.264511+00:00
[2023-11-13T10:23:11.868+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-11-13T10:23:11.969+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-11-13T10:23:12.028+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=people, task_id=save_people_to_excel, execution_date=20231113T102303, start_date=20231113T102311, end_date=20231113T102312
[2023-11-13T10:23:12.090+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-11-13T10:23:12.115+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
